dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Machines/Governors/Standard/Generic/GovCT2.mo

```diff
@@ -0,0 +1,272 @@
+  Modelica.Blocks.Math.Product prodOmegaDm annotation( Placement(visible = true, transformation(origin = {236, 180}, extent = {{8, -8}, {-8, 8}}, rotation = 180)));
+  Modelica.Blocks.Nonlinear.SlewRateLimiter rateLimitFsrt(Falling = RDownPu,Rising = RUpPu, Td = tDRatelim, initType = Modelica.Blocks.Types.Init.InitialOutput, y_start = initFsrtPu) annotation( Placement(visible = true, transformation(origin = {-4, 40}, extent = {{-8, -8}, {8, 8}}, rotation = 0)));
+  Modelica.Blocks.Nonlinear.SlewRateLimiter rateLimitPRate(Rising = PRatePu, Td = tDRatelim, initType = Modelica.Blocks.Types.Init.InitialOutput, y_start = (ValveMaxPu - WFnlPu) * KTurb)  annotation( Placement(visible = true, transformation(origin = {138, 14}, extent = {{-8, -8}, {8, 8}}, rotation = -90)));
+  Modelica.Blocks.Logical.Switch switchWFSpd annotation( Placement(visible = true, transformation(origin = {272, -126}, extent = {{10, -10}, {-10, 10}}, rotation = -90)));
+  Modelica.Blocks.Tables.CombiTable1Ds tablePLimFromf(extrapolation = Modelica.Blocks.Types.Extrapolation.LastTwoPoints, smoothness = Modelica.Blocks.Types.Smoothness.LinearSegments, table = PLimFromfPoints, tableOnFile = false, verboseRead = false) annotation( Placement(visible = true, transformation(origin = {138, 38}, extent = {{-10, -10}, {10, 10}}, rotation = -90)));
+  Modelica.Blocks.Continuous.TransferFunction transferFunctCtB(a = {tB, 1}, b = {tC, 1}, initType = Modelica.Blocks.Types.Init.InitialState, x_start = {initPMechNoLossPu}, y_start = initPMechNoLossPu) annotation( Placement(visible = true, transformation(origin = {254, 106}, extent = {{10, -10}, {-10, 10}}, rotation = -90)));
+  Modelica.Blocks.Continuous.TransferFunction transferFunctSAtSB(a = {tSB, 1}, b = {tSA, 1}, initType = Modelica.Blocks.Types.Init.InitialOutput, y_start = initTexPu) annotation( Placement(visible = true, transformation(origin = {80, 114}, extent = {{10, -10}, {-10, 10}}, rotation = 0)));
+  // initial parameters
+  parameter Types.AngularVelocityPu omega0Pu = SystemBase.omega0Pu "Initial rotor speed";
+  final parameter Types.ActivePowerPu PRef0Pu = RDroop*(if RSelectInt == 0 then 0 elseif RSelectInt == 1 then Pm0Pu else initValvePu) "Initial value of reference power";
+  parameter Types.ActivePowerPu PGen0Pu = Pm0Pu*PBaseMw/PGenBaseMw "Initial value of generated active power in pu (base SnRef (system base), generator convention";
+  parameter Types.ActivePowerPu Pm0Pu "Initial value of mechanical power";
+protected
+    // initialization helpers
+    final parameter Types.PerUnit initCfePu = WFnlPu + (if KTurb > 0 then initPMechNoLossPu/KTurb else 0) "Initial value of cfe in pu";
+    final parameter Types.PerUnit initFsrtPu = (PLdRefPu/KTurb + WFnlPu - initTexPu)*KPLoad + initIntegratorKILoadPu "Initial value of cfe in fsrt in pu";
+    final parameter Types.PerUnit initIntegratorKILoadPu = 1 "Initial value of load controller integrator in pu";
```

Comment:
It is initialized to a large value, so that it is not the active path at init.

since the limiter's upper limit is 1 (see screenshot), the largest meaningful value is 1.

![image](https://github.com/user-attachments/assets/651b648f-69d9-43d1-9012-32a48ad5aaaa)